app/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id("buildlogic.java-application-conventions")
```

Comment:
The buildlogic.java-application-conventions is part of the project and doesn't have version. It can be found at build-logic/src/main/kotlin/buildlogic.java-common-conventions.gradle.kts